test/javascript/packs/components/Punches.test.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import Punches from "../../../../app/javascript/packs/components/Punches";
+import { render, fireEvent, screen } from '@testing-library/react'
+import '@testing-library/jest-dom'
+
+
+
+
+test('renders list with worked intervals and total hours', () => {
```

Comment:
Try to follow the way how JS tests are written based on this test file: `test/javascript/packs/utils/calendar.test.js`, using `describe` and `it`.

Reply:
@mateuscelio Thanks for adding this test! The tests look good for me, I'd just suggest using `describe` and `it` as @joaoGabriel55 mentions.